runelite-client/src/main/java/net/runelite/client/plugins/groundmarkers/GroundMarkerPoint.java

```diff
@@ -25,13 +24,18 @@
  */
 package net.runelite.client.plugins.groundmarkers;
 
+import java.awt.Color;
 import lombok.Value;
+import net.runelite.api.coords.WorldPoint;
 
 @Value
 class GroundMarkerPoint
 {
-	private int regionId;
-	private int regionX;
-	private int regionY;
-	private int z;
+	private WorldPoint worldPoint;
```

Comment:
There is no reason for this worldPoint change, so unless its absolute necessary, either create your own serializer/deserializer for Guice or simply dont do this.

Reply:
I've moved encapsulation of `WorldPoint` + `Color` to a separate class and left this alone for serialization.